source/game.stationmap.bmx

```diff
@@ -3476,18 +3546,27 @@ Type TStationAntenna Extends TStationBase {_exposeToLua="selected"}
 	End Method
 
 
-	'override
-	Method GenerateGUID:String()
+	Method GenerateGUID:String() override
 		Return "station-antenna-"+id
 	End Method
 
 
-	'override
-	Method GetTypeName:String()
+	Method GetTypeName:String() override
 		Return GetLocale("STATION")
 	End Method
 
 
+	Method GetLongName:String() override {_exposeToLua}
+		Local n:String = GetName()
+		If n 
+			n = n.replace("#", "")
+			n = "#" + RSet(n, 4).Replace(" ", "0")
```

Comment:
3 sollte doch ausreichen. Und wenn es dann für einige Einträge viestellig wird, ist das auch nicht schlimm.

Reply:
Alternativvorschlag n = LSet(n,6)+Kürzel (x + Zahl ohne Padding bis zu vierstellig + Leerzeichen + Kürzel). Die Kürzel wären immer weiter untereinander, es gibt keine führenden Nullen (bei denen man sich nicht über die Anzahl einigen kann) und bei der Berechnung wären weniger replace-Operationen nötig.

Reply:
Fuer den Fall 
```
\#1    BY
\#8    BW
\#100    BW
\#103    BY
```
Sieht das aber auch doof aus ... da die 4 Leerzeichen eben immer 4 Leerzeichen sind, "n" aber verschieden breit.

Man koennte nun `DrawMapStationListEntryContent` aufbohren um das dort "schick" zu machen ... ("middleValue" einfuehren).
Fuehlt sich aber wie "Elefanten aus Muecken machen" an.

Reply:
Wenn Dir die erste Variante besser gefällt, kann ich mit der auch sehr gut leben! RSet(n, 3) sollte meiner Ansicht nach aber ausreichen:
Beim der kleinsten Radius 20 passen kaum mehr als 130 Antennen auf die Karte. Wenn jemand mit starken Überlappungen arbeitet, dürfte es dennoch lange dauern, bis 999 erreicht sind und der Spieler wegen der Kosten pro Zuschauer eher pleite geht.

In der LSet-Variante ist ein zusätzliches Leerzeichen nicht nötig (n, 6 beinhaltet schon das Zahlenpadding *und* das Leerzeichen nach einer maximal 4stelligen Zahl).

Reply:
(PS: Ich hatte es nicht ausprobiert, aber ich war davon ausgegangen, dass LSet(n,6) immer einen String mit 6 Zeichen erzeugt., also hinter \#1 kommen 4 Leerzeichen, hinter \#100 kommen 2 Leerzeichen; gefolgt vom Kürzel)

Reply:
![image](https://user-images.githubusercontent.com/2625226/174277022-adcd9883-94e6-4aa9-b2f7-91d64e55b6b6.png)

Das ist mit normalen "Freizeichen" ... nur ne Kleinigkeit, aber koennte man eventuell schoener machen :)